src/main/java/org/eclipse/xpanse/terraform/boot/api/controllers/TerraformBootFromDirectoryApi.java

```diff
@@ -96,10 +96,10 @@ public TerraformResult deployFromDirectory(
             @Valid @RequestBody
             TerraformDeployFromDirectoryRequest request,
             @RequestHeader(name = "X-Custom-RequestId", required = false) UUID uuid) {
```

Comment:
@baixinsui it is better we just use one place. Either header or request body. 
If request body is better, then let us remove the header field. We must check where is the header used already and replace that with request.

Reply:
OK. I will keep this field in the request body.